azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDKWrapper.kt

```diff
@@ -152,6 +164,31 @@ internal class ChatSDKWrapper(
         return future
     }
 
+    override fun getChatParticipants() {
+        coroutineScope.launch {
+            try {
+                val participants: MutableList<RemoteParticipantInfoModel> = mutableListOf()
+                threadClient.listParticipants().forEach {
+
+                    participants.add(
+                        RemoteParticipantInfoModel(
+                            userIdentifier = it.communicationIdentifier.into(),
+                            displayName = it.displayName
+                        )
+                    )
+                }
```

Comment:
Instead of a forEach filling a mutable list, would it make sense to write it like this instead?
```kotlin
val participants: List<RemoteParticipantInfoModel> = threadClient.listParticipants().map {
    RemoteParticipantInfoModel(
        userIdentifier = it.communicationIdentifier.into(),
        displayName = it.displayName
    )
}
```